CHANGELOG.md

```diff
@@ -7,8 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.14.0
+
 ### Breaking Changes
 
+- Minimum supported Rust version updated to 1.71
```

Comment:
`env_logger` apparently bumped their MSRV and was causing issues when built on lower versions.

Reply:
Since it's only  a dev dep, I am not sure we should bump Marv, but rather stick to `0.10.x`?

Reply:
:shrug: We certainly could.  I figured a 7 months old version was still at least reasonable